flink-python/pyflink/fn_execution/coders.py

```diff
@@ -319,7 +338,7 @@ def to_type_hint(self):
 
 @Coder.register_urn(FLINK_SCHEMA_CODER_URN, flink_fn_execution_pb2.Schema)
```

Comment:
What about moving this method to class FlattenRowCoder? It makes the code more readable.